pkg/ddevapp/ddevapp_test.go

```diff
@@ -669,6 +669,12 @@ func TestDdevXdebugEnabled(t *testing.T) {
 	runTime := util.TimeTrack(time.Now(), fmt.Sprintf("%s %s", site.Name, t.Name()))
 
 	phpVersions := nodeps.ValidPHPVersions
+
+	// arm64 builds from deb.sury.org do not have xdebug in php5.6
+	if runtime.GOARCH == "arm64" {
+		t.Log("Skipping php5.6 test on arm64 because deb.sury.org packages do not include it currently")
+		delete(phpVersions, "5.6")
+	}
```

Comment:
Silly me. This shouldn't be done in the tests, but instead in the actual definition of ValidPHPVersions on arm64. Same with Mysql and MariaDB.

Reply:
Fixed this with conditional compilation. And it even fixes the ddev binary's available choices of php, mysql, and mariadb versions.